sympy/vector/coordsysrect.py

```diff
@@ -45,10 +50,13 @@ def __new__(cls, name, location=None, rotation_matrix=None,
 
         """
 
+        global vector_name_list, variable_name_list, vector_alias, variable_alias
```

Comment:
I checked `sympy.diffgeom.CoordSystem`. I don't see how we can access base scalars or base vectors in there using the custom names passed to class contructor. Even there the names are used for printing only I think.

`global` is not necessary. I will change.